### PR TITLE
OEL-3420: Added check for rich text paragraph title.

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: templates/paragraphs/paragraph--oe-rich-text.html.twig [hunk 1]
```diff
@@ -6,7 +6,9 @@
  * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
  */
 #}
-{% include '@oe-bcl/bcl-heading/heading.html.twig' with {
-  title: content.field_oe_title,
-} only %}
+{% if content.field_oe_title is not empty %}
```
Comment (drishu) [inline templates/paragraphs/paragraph--oe-rich-text.html.twig]: use the |field_value filter to avoid render meta giving false negatives

**Review by drishu — CHANGES_REQUESTED** [review]
We can add a few lines in the paragraphs test to assert the absense of the wrapper and presence when needed.

File: templates/paragraphs/paragraph--oe-rich-text.html.twig (hunk 1 above)
Comment (tibi2303) [reply templates/paragraphs/paragraph--oe-rich-text.html.twig]: done

File: modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php [hunk 2]
```diff
@@ -46,6 +46,7 @@ abstract class ParagraphsTestBase extends AbstractKernelTestBase {
     'paragraphs',
     'text',
     'typed_link',
+    'twig_field_value',
```
Comment (drishu) [inline modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php]: remove

File: modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/RichTextTest.php [hunk 3]
```diff
@@ -108,7 +108,7 @@ public function testRendering(): void {
       'second option',
```
Comment (drishu) [inline modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/RichTextTest.php:115]: add 
```
/**
   * {@inheritdoc}
   */
  protected static $modules = [
    'twig_field_value',
  ];
```
the reason is that only this one needs it, lets start to be more efficient, the tests are slow enough

**Review by drishu — CHANGES_REQUESTED** [review]

File: modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php (hunk 2 above)
Comment (tibi2303) [reply modules/oe_whitelabel_paragraphs/tests/src/Kernel/Paragraphs/ParagraphsTestBase.php]: removed and added on the necessary tests.

**Review by drishu — APPROVED** [review]

**Pull request merged**